contracts/contracts/oracle/OracleRouter.sol

```diff
@@ -209,7 +209,25 @@ contract OETHOracleRouter is OracleRouter {
         override
         returns (address feedAddress, uint256 maxStaleness)
     {
-        if (asset == 0xD533a949740bb3306d119CC777fa900bA034cd52) {
+        if (asset == 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2) {
+            // FIXED_PRICE: WETH/ETH
+            feedAddress = FIXED_PRICE;
+            maxStaleness = 0;
+        } else if (asset == 0x5E8422345238F34275888049021821E8E08CAa1f) {
+            // frxETH/ETH
+            feedAddress = 0xC58F3385FBc1C8AD2c0C9a061D7c13b141D7A5Df;
+            maxStaleness = 1 days;
```

Comment:
1 day seems right. Maybe a comment that this currently updates every 18 hours.

Reply:
I missed that all the others have a `STALENESS_BUFFER` of 1 day added to their `maxStaleness`. Should we do that same for frxETH/ETH?

Reply:
The other oracles have a 24 hour heartbeat, that they sometimes exceed, thus the buffer. The fraxETH oracle has an 18 hour heartbeat.

So we could just write 18 hours + buffer, to be explicit, or make a comment on it.

I didn't realize our buffer was a day, so I guess it would make more sense to do expected time + buffer rather than a comment.

Reply:
thanks. I've pushed a change to 18 hours + STALENESS_BUFFER